From 33e50dafd4fe9e16e0627869a2fe23453c06b788 Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Mon, 31 Oct 2022 12:28:40 +0800
Subject: [PATCH] Revert "Linux 5.10.152"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 7d51b4c67cfb95a069ccbe52f13963bfd9fe85b0.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index a0750d051982..8d63b1bff690 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 152
+SUBLEVEL = 151
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.3

